beacon/sync/src/main/java/tech/pegasys/teku/beacon/sync/forward/singlepeer/FailedBlobSidecarImportException.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright ConsenSys Software Inc., 2023
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.beacon.sync.forward.singlepeer;
+
+import tech.pegasys.teku.spec.datastructures.execution.versions.deneb.BlobSidecar;
+
+public class FailedBlobSidecarImportException extends RuntimeException {
```

Comment:
why not extending `InvalidResponseException` like `FailedBlockImportException` ?

Reply:
We could also add the import result for better errors handling in `handleFailedRequestToPeer`

Reply:
Agree about extending the `InvalidResponseException`. Currently, the interface for importing a sidecar returns Void, if we change in the future to sth like `BlobSidecarImportResult` then we can change the exception class a little bit and make it nicer.